### ISSUE TITLE
How to recognize a custom scene view image?

### ISSUE THREAD

**Issue by tangshi** [potterhsu/SVHNClassifier#1]

**Comment by seovchinnikov** [comment]
I think its easy to convert it to matlab format or you can write conerter to tfrecords directly

**Comment by MarStarck** [comment]
prepare your images and ground-truth labels and then fill the tfrecords file format
```
example = tf.train.Example(features=tf.train.Features(feature={
            'image': ExampleReader._bytes_feature(image),
            'length': ExampleReader._int64_feature(length),
            'digits': tf.train.Feature(int64_list=tf.train.Int64List(value=digits))
        }))
```
